port.h

```diff
@@ -116,7 +116,15 @@ SOFTWARE.
 #define M_PI 3.14159265359
 #endif
 
+#endif  // CURRENT_WINDOWS
+
+#ifdef CURRENT_APPLE
+#ifdef CURRENT_APPLE_ENABLE_NSLOG
+#define CURRENT_NSLOG(...) ::NSLog(__VA_ARGS__)
+#else
+void CURRENT_NSLOG(...) {}
```

Comment:
Empty define may cause warnings, if the argument is used inly for logging.
What's wrong with global no-op function? I didn't get the idea with semicolon.